vendor/engines/saas/app/jobs/base_login_job.rb

```diff
@@ -0,0 +1,36 @@
+class LoginJob < ActiveJob::Base
+
+  def perform(params)
+    payload = map_payload(params)
+    Analytics::IdentifyUserJob.perform_later(payload)
+
+    #email = map_email(params)
+    #Queue job to send the email
+  end
+
+  def self.action(action)
+    @action = action
+  end
```

Comment:
I remember having trouble subclassing class attributes 

Why aren't you using the class_attr stuff here?